src/rdkafka_admin.c

```diff
@@ -7427,6 +7585,8 @@ rd_kafka_DescribeConsumerGroupsResponse_parse(rd_kafka_op_t *rko_req,
         rd_kafka_error_t *error   = NULL;
         char *group_id = NULL, *group_state = NULL, *proto_type = NULL,
              *proto = NULL, *host = NULL;
+        rd_kafka_AclOperation_t *operations = NULL;
+        int operation_cnt;
```

Comment:
initialize to 0 in case authorized operations list isn't supported

Reply:
I've initialized to -1 for that case. It is further handled by rd_kafka_ConsumerGroupDescription_new, that's why I've done so